lib/bamboo/phoenix.ex

```diff
@@ -122,7 +122,7 @@ defmodule Bamboo.Phoenix do
 
       def text_email_layout do
         new_email
-        # Will use MyApp.LayoutView with email.text layout when rendering text emails
+        # Will use MyApp.LayoutView with email.text template when rendering text emails
```

Comment:
Actually, sorry, just saw that its actually broken down to `{layout, template}`, so yes, 👍  on the change